src/main/java/core/basesyntax/model/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.model;
+
+public class IsoscelesTrapezoid extends Color {
```

Comment:
The class IsoscelesTrapezoid should not extend Color. Instead, it should have a color property if needed. Consider using composition over inheritance here.